### PR TITLE
Including amenities and places to hashtag endpoint

### PR DESCRIPTION
**Pull request by ramyaragupathy**
Modified the base model, query builder and test function to include new amenities and places statistics.

cc @itskshitiz321

### CODE REVIEW

File: tests/src/test_app.py [hunk 1]
```diff
@@ -427,7 +427,8 @@ def test_organization_hashtag_weekly_query():
             where name = 'msf'
             ),
             t2 as (
-                select name as hashtag, type as frequency , start_date , end_date , total_new_buildings , total_uq_contributors as total_unique_contributors , total_new_road_m as total_new_road_meters
```
Comment (itskshitiz321) [inline tests/src/test_app.py:430]: Update will be needed on Assertion Step ! Replacement of existing query with updated query that will be generated from query builder

File: tests/src/test_app.py [hunk 2]
```diff
@@ -455,7 +456,8 @@ def test_organization_hashtag_monthly_query():
...
+                select name as hashtag, type as frequency , start_date , end_date , total_new_buildings , total_uq_contributors as total_unique_contributors , total_new_road_m as total_new_road_meters,
```
Comment (itskshitiz321) [inline tests/src/test_app.py:459]: Same here

File: tests/src/test_app.py (hunk 1 above)
Comment (ramyaragupathy) [reply tests/src/test_app.py:473]: Fixed through https://github.com/hotosm/galaxy-api/pull/178/commits/3e963b3f63cd84b0f831b35bf97262c4547342cb and https://github.com/hotosm/galaxy-api/pull/178/commits/3c24122c9519a2891687b7f8db0b9e486208b6f4

File: tests/src/test_app.py (hunk 2 above)
Comment (ramyaragupathy) [reply tests/src/test_app.py:502]: Fixed through https://github.com/hotosm/galaxy-api/pull/178/commits/3e963b3f63cd84b0f831b35bf97262c4547342cb and https://github.com/hotosm/galaxy-api/pull/178/commits/3c24122c9519a2891687b7f8db0b9e486208b6f4

**Comment by ramyaragupathy** [comment]
@itskshitiz321 - please check now

**Pull request merged**